modules/gitops-release-notes-1-12-2.adoc

```diff
@@ -0,0 +1,70 @@
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-12-2_{context}"]
+= Release notes for {gitops-title} 1.12.2
+
+{gitops-title} 1.12.2 is now available on {OCP} 4.12, 4.13, 4.14, and 4.15.
+
+[id="errata-updates-1-12.2_{context}"]
+== Errata updates
+
+[id="RHSA-2024:130830-04-gitops-1-12-2-security-update-advisory_{context}"]
+=== RHSA-2024:130830-04 - {gitops-title} 1.12.2 security update advisory
+
+Issued: 2024-05-09
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2024:130830-04[RHSA-2024:130830-04]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="new-features-1-12-2_{context}"]
+== New features
+
+* With this update, support has been provided for the `must-gather` tool in the Argo Rollouts Operator. This update provides support for the following enhancements:
+
+** Logs for Argo Rollouts Operator pods and Argo Rollouts pods.
+** Contents of Argo Rollouts Manager and Argo Rollouts custom resources (CRs).
+** Contents of `Deployment`, `Statefulset`, `ConfigMaps` CRs created by the `must-gather` tool in the Argo Rollouts Operator.
+link:https://issues.redhat.com/browse/GITOPS-3947[GITOPS-3947]
+
+[id="fixed-issues-1-12-2_{context}"]
+== Fixed issues
+
+* Before this update, users could not use the `argocd-k8s-auth` binary to add GKE and EKS clusters because this binary was not available in the {gitops-shortname} container. This update fixes the issue by adding the `argocd-k8s-auth` binary in the {gitops-shortname} container. link:https://issues.redhat.com/browse/GITOPS-4226[GITOPS-4226]
+
+* Before this update, attempts to connect to Azure DevOps with Argo CD would result in an error due to the deprecation of the `rsa-ssh` host key algorithm by the Azure DevOps Repository service. This update fixes the issue by providing support for the `rsa-ssh` host key algorithms during the communication process between Argo CD and Azure DevOps Repository service. link:https://issues.redhat.com/browse/GITOPS-4543[GITOPS-4543]
+
+* Before this update, {gitops-shortname} console plugin workloads did not schedule on infrastructure nodes when the `runOnInfra` field was enabled in the `GitOpsService` custom resource (CR). This update fixes the issue by adding the infrastructure `node-selector` on the {gitops-shortname} console plugin workloads. This enables users to configure custom node-selectors and tolerations on the `gitops-console` pod. As a result, when the `runOnInfra` field is enabled, the {gitops-shortname} console plugin pod is placed on the infrastructure nodes similarly to the other default workloads. link:https://issues.redhat.com/browse/GITOPS-4496[GITOPS-4496]
```

Comment:
```suggestion
* Before this update, {gitops-shortname} console plugin workloads did not schedule on infrastructure nodes when the `runOnInfra` field was enabled in the `GitOpsService` custom resource (CR). This update fixes the issue by adding the infrastructure `node-selector` on the {gitops-shortname} console plugin workloads. This enables users to configure custom node-selectors and tolerations on the `gitops-console` pod. As a result, when the `runOnInfra` field is enabled, the {gitops-shortname} console plugin pod is placed on the infrastructure nodes like the other default workloads. link:https://issues.redhat.com/browse/GITOPS-4496[GITOPS-4496]
```
Hello @Dhruv-Soni11 we discussed this with Sri and this wording can have double meaning, so changing it to simpler word is the best. Thank You!

Reply:
Fixed.